packages/portal/src/pages/item/_.vue

```diff
@@ -356,27 +357,207 @@
       },
 
       async fetchMetadata() {
+        const params = {};
+        if (this.translatingMetadata) {
+          params.profile = 'translate';
+          params.lang = this.metadataLanguage;
+        }
+
+        let data;
         try {
-          const response = await this.$apis.record.getRecord(
-            this.identifier,
-            { locale: this.$i18n.locale, metadataLanguage: this.$auth.loggedIn ? this.$route.query.lang : undefined }
-          );
+          data = await this.$apis.record.get(this.identifier, params);
+        } catch (error) {
+          const errorResponse = error.response;
+          if (errorResponse?.status === 502 && errorResponse?.data?.code === '502-TS' && !this.fromTranslationError) {
+            this.fromTranslationError = true;
+            data = await this.$apis.record.get(this.identifier);
+          } else {
+            return this.$error(error, { scope: 'item' });
+          }
+        }
+        const edm = data.object;
+
+        if (this.identifier !== edm.about) {
+          return this.redirectToAltRoute({ params: { pathMatch: edm.about?.slice(1) } });
+        }
+
+        this.identifier = edm.about;
+        this.type = edm.type;
+
+        const item = new Item(edm);
+
+        this.allMediaUris = item.providerAggregation.displayableWebResources.map((wr) => wr.about);
+        this.iiifPresentationManifest = item.iiifPresentationManifest;
+        this.isShownAt = item.providerAggregation.edmIsShownAt;
+        this.media = item.providerAggregation.displayableWebResources;
+
+        this.entities = this.extractEntities(edm);
+
+        this.metadata = this.extractMetadata(edm);
+
+        process.client && this.trackCustomDimensions();
+      },
+
+      forEachLangMapValue(langMapContainer, callback) {
+        for (const field in langMapContainer) {
+          if (isLangMap(langMapContainer[field])) {
+            for (const locale in langMapContainer[field]) {
+              callback(field, locale);
+            }
+          }
+        }
+      },
+
+      findProxy(proxies, type) {
+        return proxies.find(proxy => proxy.about?.startsWith(`/proxy/${type}/`));
+      },
+
+      extractEntities(edm) {
+        return [].concat(edm.concepts, edm.places, edm.agents, edm.timespans, edm.organizations)
+          .filter(Boolean)
+          .map((entity) => pick(entity, [
+            'about',
+            'latitude',
+            'longitude',
+            'prefLabel'
+          ]))
+          .map((entity) => reduceLangMapsForLocale(entity, this.metadataLanguage));
+      },
+
+      extractEuropeanaProxy(edm) {
+        const europeanaProxy = this.findProxy(edm.proxies, 'europeana') || {};
+
+        // Europeana proxy only really needed for the translate profile
+        if (this.translatingMetadata) {
+          for (const field in europeanaProxy) {
+            if (europeanaProxy[field][this.metadataLanguage]) {
```

Comment:
Should this be `translationLanguage` instead of `metadataLanguage`?

Reply:
They will be the same value if translatingMetadata is true, so either will work here.